cmd/cli/plugin/cluster/set_node_pool.go

```diff
@@ -32,6 +33,7 @@ var clusterSetNodePoolCmd = &cobra.Command{
 func init() {
 	clusterSetNodePoolCmd.Flags().StringVarP(&setNodePoolOptions.FilePath, "file", "f", "", "The file describing the node pool (required)")
 	clusterSetNodePoolCmd.Flags().StringVar(&setNodePoolOptions.Namespace, "namespace", "default", "The namespace the cluster is found in.")
+	clusterSetNodePoolCmd.Flags().StringVar(&setNodePoolOptions.MachineDeploymentBase, "machine-deployment-base", "", "The machine deployment to use as a base for creating a new node pool")
```

Comment:
We should add in help that machine-deployment-base option is not applicable to TKGS